run_portal.ps1

```diff
@@ -0,0 +1,41 @@
+# Change to uid2-admin directory, and push location to stack
+Push-Location -Path "../uid2-admin" -ErrorAction Stop
+
+# Run admin docker & application
+try {
+    Write-Host "Running docker compose for uid2-admin..."
+    docker compose up -d
+    if ($LASTEXITCODE -ne 0) {
+        Write-Error "Docker compose failed in uid2-admin with exit code $LASTEXITCODE"
+        exit 1
+    }
+
+    Write-Host "Running admin application..."
+    Start-Process "mvn" -ArgumentList "clean compile exec:java", "-Dvertx-config-path=conf/local-config.json", "-Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.SLF4JLogDelegateFactory", "-Dlogback.configurationFile=conf/logback.xml" -WindowStyle Hidden
```

Comment:
Start-Process runs the command in a separate terminal, allow the original terminal to continue exiting. I've decided to hide the separate terminal using -WindowStyle Hidden because it doesn't really add much value and is yet another terminal.

This behaviour ends up being similar to docker's detached (-d) mode, which is used throughout this script.

Reply:
There's no point in checking for the $LASTEXITCODE after this line, as it is run asynchronously so it won't return an exit code here anyway.